dataRetriever/dataPool/proofsCache/proofsCache_bench_test.go

```diff
@@ -0,0 +1,66 @@
+package proofscache_test
```

Comment:
If we move the test to package `proofscache`, maybe we can drop some code from the `export_test` file. Can also stay as it is.

Reply:
i would keep it in _test package since there are other useful functions from test

Reply:
and we plan to make this component more generic and to export it in other packages as well (for headers pool)